server/index.js

```diff
@@ -1,9 +1,21 @@
-const express = require('express');
+import express from 'express';
+import connectDB from './config/db.js';
+import bodyParser from 'body-parser';
+import dotenv from 'dotenv';
+dotenv.config();
+
+connectDB();
 
 const app = express();
 
-app.get('/', (req, res) => {
-    res.send('<h1>Hello world!</h1>');
+//Middleware
+app.use(bodyParser.json());
+app.use(bodyParser.urlencoded({ extended: true }));
```

Comment:
I think instead of body-parser, I think we can do all this with express (without bodyParser)
for example:
app.use(express.json());
app.use(express.urlencoded({ extended: false }));

Can you confirm this pls? If this works, we can remove body-parser (and uninstall it) :)

Reply:
it works